actinia_stac_plugin/api/stac.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """
-Copyright (c) 2021 mundialis GmbH & Co. KG
+Copyright (c) 2022 mundialis GmbH & Co. KG
```

Comment:
The two copyright dates need to match..
```suggestion
Copyright (c) 2018-2022 mundialis GmbH & Co. KG
```